.buildkite/scripts/packer_cache.sh

```diff
@@ -13,3 +13,6 @@ yarn kbn bootstrap
 for version in $(cat versions.json | jq -r '.versions[].version'); do
   node scripts/es snapshot --download-only --base-path "$ES_CACHE_DIR" --version "$version"
 done
+
+echo "--- Cloning repos for docs build"
+node scripts/validate_next_docs --clone-only
```

Comment:
@brianseeders do you know if we will have the keys/tokens necessary to clone private repos at this point? I assume so

Reply:
Off the top of my head, I don't think so. Those are pulled at runtime in a hook, and I don't believe the packer build currently requires anything. It shouldn't be too hard to make it happen, it just doesn't at the moment

Reply:
I think this should handle it.